Sources/Shared/UI/Documents/DocumentsViewController.m

```diff
@@ -243,6 +242,8 @@ - (void)dealloc
     [_tblFiles release];
     _tblFiles = nil;
     
+    [_popoverProperties release];
```

Comment:
Surround this by 
```
if (_popoverProperties) 
  [_popoverProperties release];
```
No need for `_popoverProperties = nil` since the variable is set just after.